lib/iris/experimental/regrid_conservative.py

```diff
@@ -329,16 +331,23 @@ def _valid_units(coord):
 
     # Return result as a new cube based on the source.
     # TODO: please tidy this interface !!!
-    return RectilinearRegridder._create_cube(
-        fullcube_data,
-        src=source_cube,
-        x_dim=src_dims_xy[0],
-        y_dim=src_dims_xy[1],
+    _regrid_callback = functools.partial(
```

Comment:
In this module it says we have deprecated `iris.experimental.regrid_conservative` so this change is probably unnecessary?